fe/fe-core/src/test/java/org/apache/doris/nereids/trees/plans/PlanEqualsTest.java

```diff
@@ -193,7 +195,12 @@ public void testPhysicalOlapScan(
             @Mocked DistributionSpecHash distributionSpecHash) {
         List<String> qualifier = Lists.newArrayList();
 
-        PhysicalOlapScan olapScan = new PhysicalOlapScan(olapTable, qualifier, distributionSpecHash, Optional.empty(),
+        ArrayList<Long> selectedTabletId = Lists.newArrayList();
+        for (Partition partition : olapTable.getAllPartitions()) {
+            selectedTabletId.addAll(partition.getBaseIndex().getTabletIdsInOrder());
+        }
+        PhysicalOlapScan olapScan = new PhysicalOlapScan(olapTable, qualifier, olapTable.getBaseIndexId(),
+                selectedTabletId, olapTable.getPartitionIds(), distributionSpecHash, Optional.empty(),
                 logicalProperties);
 
         Assertions.assertEquals(olapScan, olapScan);
```

Comment:
compare same object is not a good idea, it is better to new two different object and then compare them